UnitedIncome/components
# Adds isDark prop to the ChanceOfSuccess component

### Description
> Please provide a brief description of your changes below.

Adds the option for the text in the chance of success meter to be dark. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Click `isDark`
* It will be dark!
### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
